Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserTreeToTableProxyModel.cpp

```diff
@@ -268,9 +272,10 @@ namespace AzToolsFramework
 
         void AssetBrowserTreeToTableProxyModel::RowsInserted(const QModelIndex& parent, int start, int end)
         {
-            AZ_Assert(sourceModel()->index(start, 0, parent).isValid(), "Index invalid");
+            auto sourceModelPtr = sourceModel();
```

Comment:
Probably don't need "Ptr" in the name? Kinda hungarian notation.
```suggestion
QAbstractItemModel* sourceModel = sourceModel();
```

Reply:
I mainly did that so that the variable did not have the same name as the function. I can abbreviate it or decorate it with anything to make it different.